### PR TITLE
feat: verdaccio for acceptance test

### PR DESCRIPTION
**Pull request by juanpicado**
**What's the problem this PR addresses?**

Current *acceptance-test* are based on a mocked registry and each new scenario might need to modify that server and increase complexity, more code to maintain and less intuitive configuration.

**How did you fix it?**

The proposal is to use Verdaccio instead as a registry server.  There are a few advantages of this migration:

-  No need to modify `pkg-tests-fixtures` at all. All fixtures packages will be published in the setup phase.
-  Include custom authentification plugin to make much easier the current auth model.

What's missing? This is just a wip PoC (just limited to one test file) with the idea to discover challenges

- [x] Add verdaccio as a dependency
- [x] Add custom configuration
- [ ] Pre publish fixtures packages: This is required otherwise verdaccio won't be able to find any package
- [ ] Enable all tests
- [ ] Add test documentation
- [ ] Clean up (a bit messy now)

I creating this as a draft because still too much work to do, but I just wanna share current progress in order to get early feedback (and I need the CI to verify all works properly).

### CODE REVIEW

File: packages/acceptance-tests/pkg-tests-core/sources/utils/config.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+storage: /Users/jpicado/.local/share/verdaccio/storage_yarn
```
Comment (arcanis) [inline packages/acceptance-tests/pkg-tests-core/sources/utils/config.yaml]: I guess this won't work everywhere :p

**Comment by juanpicado** [comment]
We can add a middleware plugin adding new endpoints and that might work.

File: packages/acceptance-tests/pkg-tests-core/sources/utils/config.yaml (hunk 1 above)
Comment (juanpicado) [reply packages/acceptance-tests/pkg-tests-core/sources/utils/config.yaml]: Hehe ... yeah the idea is create a temp folder and publish all fixture packages there on the setup phase. For now I am working in that part, I managed to publish but I don’t know how to invoke ‘yarn publish’ programmatically with the dev version. I did it using exec with npm ☺️) just for proving works my approach.

**Comment by juanpicado** [comment]
@arcanis I'll close this, I'm having no much time lately and also there is a bug with the Node API which I was aiming to use here. I'll try again when I get more time.

**Pull request closed without merging**